app/helpers/quadicon_helper.rb

```diff
@@ -216,6 +221,48 @@ def render_quadicon_label(item, row)
     end
   end
 
+  # Renders a quadicon for PhysicalServer
+  #
+  def render_physical_server_quadicon(item, options)
+    output = []
+    if settings(:quadicons, :physical_server)
+      output << flobj_img_simple("layout/base.png")
+
+      output << flobj_p_simple("a72", (item.host ? 1 : 0))
+      output << flobj_img_simple("svg/currentstate-#{h(item.power_state.downcase)}.svg", "b72")
+      output << flobj_img_simple(img_for_physical_vendor(item), "c72")
+      output << flobj_img_simple(img_for_health_state(item), "d72")
+      output << flobj_img_simple('100/shield.png', "g72") unless item.get_policies.empty?
+    else
+      output << flobj_img_simple(size)
+      output << flobj_img_simple(width * 1.8, img_for_physical_vendor(item), "e72")
+    end
+
+    if options[:typ] == :listnav
+      # Listnav, no href needed
+      output << content_tag(:div, :class => 'flobj') do
+        tag(:img, :src => ActionController::Base.helpers.image_path("layout/reflection.png"), :border => 0)
```

Comment:
I know `tag(:img)` exists elsewhere in this code, but I don't think it's necessary. Should be able to use `quadicon_reflection_img` here. Plus, I think doing this will clear up the CodeClimate complaint.